_pages/backup-restore.md

```diff
@@ -25,6 +25,19 @@ To take the backup:
 1. Once the backup is complete, refresh the *Backups* list to check it is complete (it may take some time) - note that the backup process is run as a Skyve *Job* and the progress and completion of the backup can be reviewed from the admin *Jobs* menu
 1. If you wish to download the backup, select the backup from the list and press *Download Backup*
 
+### Backup options
+Before making a backup, there are options found next to the *Backup* button, highlighted below:
+
+- **Content** - Check this to include content in the backup that is made
+- **Audit Log** - Check this to include the audit log that are made by the system into the backup created.
```

Comment:
Describe what the audit log is (record of changes in the system) and that it's on by default